circle.yml

```diff
@@ -1,3 +1,18 @@
 dependencies:
   pre:
     - sudo add-apt-repository -y ppa:ethereum/ethereum; sudo apt-get update; sudo apt-get install -y solc
+  override:
+    - echo "OK, let's go!"
```

Comment:
@rtkaczyk Can you leave a comment, at least here, why did you need to put this? I recall you mentioned something but imo it's better if it's written somewhere :wink: